packages/core/src/actions/network-status/blockNumber.ts

```diff
@@ -0,0 +1,12 @@
+import { WithProvider } from '../../types'
+
+export type BlockNumberActionArgs = WithProvider<unknown>
```

Comment:
`unknown` here works but is a bit gross. Open to alternatives since ts complains on `{}`.